package.json

```diff
@@ -63,6 +63,31 @@
   ],
   "main": "./dist/extension",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "workspaceViewer",
+          "title": "R Workspace",
+          "icon": "./images/Rlogo.svg"
+        }
+      ]
+    },
+    "views": {
+      "workspaceViewer": [
+        {
+          "id": "workspaceViewer",
+          "name": "R Workspace",
```

Comment:
This name seems a bit redundant as I move the R icon  from the side bar to one of the views, it becomes

<img width="287" alt="image" src="https://user-images.githubusercontent.com/4662568/100545397-9a4aab00-3296-11eb-9920-b3b84f98db64.png">

Reply:
Maybe the `activitybar` should take a title `R`, and `workspaceViewer` has the name `Workspace`?

Reply:
Solved in c3f3c17 

![image](https://user-images.githubusercontent.com/60372411/100562521-e821dc00-32b3-11eb-9e2b-353f1ca8d867.png)